src/database/seeding/seeding.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { Controller, Get, Post, Body, Patch, Param, Delete } from '@nestjs/common';
+import { SeedingService } from './seeding.service';
+import { CreateSeedingDto } from './dto/create-seeding.dto';
+import { UpdateSeedingDto } from './dto/update-seeding.dto';
+
```

Comment:
Get rid of this. You don't need a full CRUD api for seeding data.

Seed the requested data once and only once, when the database is empty.

And just create a dummy endpoint for returning the seeded data.